FileShareAdminClient/FileShareApiAdminClient.cs

```diff
@@ -20,21 +21,24 @@ namespace UKHO.FileShareAdminClient
     public interface IFileShareApiAdminClient : IFileShareApiClient
     {
         Task<IResult<AppendAclResponse>> AppendAclAsync(string batchId, Acl acl, CancellationToken cancellationToken = default);
-        Task<IBatchHandle> CreateBatchAsync(BatchModel batchModel, CancellationToken cancellationToken = default);
+        Task<IBatchHandle> CreateBatchAsync(BatchModel batchModel);
+        Task<IResult<IBatchHandle>> CreateBatchAsync(BatchModel batchModel, CancellationToken cancellationToken = default);
```

Comment:
No point in adding the default value for the CancellationToken, as the method overload on the previous line will prevent it from being used.

```suggestion
        Task<IResult<IBatchHandle>> CreateBatchAsync(BatchModel batchModel, CancellationToken cancellationToken);

Reply:
Removed the the default value for the CancellationToken for CreateBatchAsync and AddFileToBatch methods.